src/components/split-button/split-button.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, select } from '@storybook/addon-knobs';
+import SplitButton from './split-button';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Button from '../button';
+import notes from './notes.md';
+
+storiesOf('Split Button', module).add(
+  'default',
+  () => {
+    const as = select('as', OptionsHelper.themesBinary, OptionsHelper.themesBinary[0]);
+    const dataElement = text('data-element');
+    const dataRole = text('data-role');
+    const disabled = boolean('disabled', false);
```

Comment:
Can we take the default value from the component? I.e. `SplitButton.defaultProps.disabled`